crates/runc-shim/src/task.rs

```diff
@@ -95,15 +95,27 @@ where
 }
 
 impl<F, C> TaskService<F, C> {
-    pub async fn get_container(&self, id: &str) -> TtrpcResult<MappedMutexGuard<'_, C>> {
-        let mut containers = self.containers.lock().await;
+    pub async fn get_container_write(&self, id: &str) -> TtrpcResult<RwLockMappedWriteGuard<'_, C>> {
```

Comment:
I'd suggest to name this `container` and `container_mut` to be closed to Rust's naming conventions.

Reply:
I've changed, but I don't know , if we can give a timeout in read_lock? If some write lock is hang on , but read lock can return a error?